src/instr.rs

```diff
@@ -1,23 +1,76 @@
 use parser;
+use edit::EditMode;
+use edit::ViMode;
 
 pub enum Instr {
```

Comment:
I considered this, but some of the operations are identical in both modes. To avoid duplicating these implementations, I just put all of the operations together.                                                                                                                                                                                                                                                                                                                                        -- Matt                                                                                                                                                                                                                From: Sam RijsSent: Friday, December 18, 2015 8:57 PMTo: srijs/rust-copperlineReply To: srijs/rust-copperlineCc: codemakerSubject: Re: [rust-copperline] add vi mode support (#5)In src/instr.rs:
> @@ -1,23 +1,76 @@
>  use parser;
> +use edit::EditMode;
> +use edit::ViMode;
>  
>  pub enum Instr {

To keep the interpretation logic simple, it might make sense to come up with different instruction sets per mode. But I'm happy to merge your changes with a joint instruction set and take it from there.

—Reply to this email directly or view it on GitHub.